src/mono/mono/metadata/qcalllist.h

```diff
@@ -0,0 +1,37 @@
+#ifndef _MONO_METADATA_QCALLLIST_H_
+#define _MONO_METADATA_QCALLLIST_H_
+
+extern const void* gPalGlobalizationNative[];
+extern const void* gEventPipeInternalFuncs[];
+
+
+typedef struct MonoQCallDef
+{
+    const char* class_name;
+    const char* namespace_name;
+    const void**  functions;
+} MonoQCallDef;
+
+typedef struct MonoQCallFunc {
+    int*            flags;
+    void*           implementation;
+    const char*     method_name;
+} MonoQCallFunc;
+
+enum {
+    func_flag_end_of_array   = 0x01,
+    func_flag_has_signature = 0x02,
+    func_flag_unreferenced = 0x04, // Suppress unused fcall check
+    func_flag_qcall        = 0x08, // QCall - mscorlib.dll to mscorwks.dll transition implemented as PInvoke
+};
+
+const MonoQCallDef c_qcalls[] =
```

Comment:
It's strange to see an array definition in a header in C.  (i'm not sure, but we will get multiple copies if this header is included in different .c files, I think)

Can we follow the icall model a bit here:
1. make this array static to `native-library-qcall.c`
2. make a `qcall-def.h` header that just has `FCClassElement`s one per line
3. Something like this in native-library-qcall.c
   ```c
    static const c_qcalls[] = {
    #define FCClassElement(name,namespace,funcs) {name, namespace, funcs},
    #inlcude "qcalllist-def.h"
    #undef FCClassElement
    };
   ```

I'm not super thrilled about the `gEventPipeInternalFuncs` and `gPalGlobalizationNative` globals either.  Do they need to be global?  if so, we should use a `mono_` prefix on them